packages/core/parcel-bundler/src/Bundler.js

```diff
@@ -135,7 +135,8 @@ class Bundler extends EventEmitter {
       detailedReport: options.detailedReport || false,
       global: options.global,
       autoinstall:
-        typeof options.autoinstall === 'boolean'
+        typeof options.autoinstall === 'boolean' &&
+        process.env.PARCEL_AUTOINSTALL !== 'false'
```

Comment:
I think the local `autoinstall` option should override the environment variable. The environment variable should only apply if the option is not set.